_dcp/data/provider/jboss-developer.json

```diff
@@ -427,7 +427,7 @@
       "description":"JBoss Developer website pages",
       "sys_type":"webpage",
       "input_preprocessors": [
-        { 
+        {
```

Comment:
Try not to make white-space changes in un-changed areas of code. It makes it harder to detect what material changes you have made, when reviewing the PR. I think it would probably also increase the chances of merge conflicts.